docs/source/markdown/options/decryption-key.md

```diff
@@ -0,0 +1,7 @@
+####> This option file is used in:
+####>   podman create, pull, run
+####> If you edit this file, make sure your changes
+####> are applicable to all of those.
```

Comment:
```suggestion
####> are applicable to all of those commands.
```

Reply:
If you see any other file in `docs/source/markdown/options/`, they all have the last sentence as "are applicable to all of those.". That is, they don't have "commands" as the last word. So to maintain consistency with the rest of `.md` files, I have followed their suite. Do you suggest to make it different only in `decryption-key.md`?

Reply:
@gupttaru you can ignore this comment of mine.  I've been told since this is all autogenerated.  I would like to see it get changed, but nothing to worry about for this particular PR.